frontstage/controllers/party_controller.py

```diff
@@ -154,3 +176,40 @@ def verify_token(token):
         raise ApiError(logger, response, log_level=log_level, message='Failed to verify token', token=token)
 
     logger.debug('Successfully verified token')
+
+
+def get_enrolment_details(business_party_id, enrolments):
```

Comment:
Doesn't appear to be any unit test coverage of these: 

- get_enrolment_details
- get_respondent_enrolments
- get_survey_list_details_for_party